addons/zeusHC/functions/fnc_createEntityDismounts.sqf

```diff
@@ -0,0 +1,81 @@
+/*
+ * Author: PabstMirror
+ * Creates a dismount squad arround a vehicle and adds parachutes for units exiting aircraft
+ */
+#include "script_component.hpp"
+
+params ["_attachedVehicle", "_newGroup", "_createUnits", "_placerOwner"];
+TRACE_4("params",_attachedVehicle, _newGroup, _createUnits,_placerOwner);
+
+
+//Exit if bad conditions and send hint back to orginal module placer:
+if (isNull _attachedVehicle) exitWith {
+    diag_log text format ["[POTATO] - Attachable not attached"];
+    ["Module needs to be attached on a vehicle"] remoteExecCall [QACEFUNC(common,displayTextStructured), _placerOwner];
+};
+if ((_attachedVehicle isKindOf "Air") && {((getPosATL _attachedVehicle) select 2) < 75}) exitWith {
+    diag_log text format ["[POTATO] - Air veh too low"];
+    ["Aircraft is too low"] remoteExecCall [QACEFUNC(common,displayTextStructured), _placerOwner];
+};
+if ((!(_attachedVehicle isKindOf "Air")) && {(vectorMagnitude velocity _attachedVehicle) > 5}) exitWith {
+    diag_log text format ["[POTATO] - Ground veh too fast"];
+    ["Vehicle moving too fast"] remoteExecCall [QACEFUNC(common,displayTextStructured), _placerOwner];
+};
+
+
+private _cargoSelection = getText (configFile >> "CfgVehicles" >> (typeOf _attachedVehicle) >> "memoryPointsGetInCargo");
+if (_cargoSelection == "") then {
+    private _cargoSelection = getArray (configFile >> "CfgVehicles" >> (typeOf _attachedVehicle) >> "memoryPointsGetInCargo");
+    if (_cargoSelection isEqualTo []) exitWith {};
```

Comment:
We should show a feedback message here, telling Zeus the vehicle doesn't support this feature.